docs/server/source/quickstart.md

```diff
@@ -6,34 +6,55 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 [Install MongoDB Server 3.4+](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
 
-B. Run MongoDB. Open a Terminal and run the command:
+Default database directory where MongoDB stores its data: /data/db.
```

Comment:
This line can be removed because step B. already notes that /data/db is the default database path.